From dd6771cdf6f04cde37eb14891573c0e55be83241 Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Sun, 26 Apr 2015 17:53:15 +0200
Subject: [PATCH] configure, xpresent.pc: require xext, xfixes and xrandr

include/X11/extensions/Xpresent.h:#include <X11/extensions/Xfixes.h>
include/X11/extensions/Xpresent.h:#include <X11/extensions/Xrandr.h>
include/X11/extensions/Xpresent.h:#include <X11/extensions/sync.h>

v2: also add requirement in xpresent.pc since we include those in our
    public header

Signed-off-by: Julien Cristau <jcristau@debian.org>
---
 configure.ac   | 2 +-
 xpresent.pc.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index a88ca7f..bb0939d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -58,7 +58,7 @@ PRESENTEXT_VERSION=[`echo $VERSION | sed 's/^\([0-9][0-9]*\.[0-9][0-9]*\).*$/\1/
 AC_SUBST(PRESENTEXT_VERSION)
 
 # Obtain compiler/linker options for depedencies
-PKG_CHECK_MODULES(PRESENTEXT, xproto [presentproto >= $PRESENTEXT_VERSION] xextproto x11)
+PKG_CHECK_MODULES(PRESENTEXT, xproto [presentproto >= $PRESENTEXT_VERSION] xextproto x11 xext xfixes xrandr)
 
 AC_CONFIG_FILES([Makefile
 		src/Makefile
diff --git a/xpresent.pc.in b/xpresent.pc.in
index d5d0d38..8f6047a 100644
--- a/xpresent.pc.in
+++ b/xpresent.pc.in
@@ -7,6 +7,6 @@ Name: Xpresent
 Description: X Present  Library
 Version: @PACKAGE_VERSION@
 Requires: xproto presentproto >= @PRESENTEXT_VERSION@
-Requires.private: x11
+Requires.private: x11 xext xfixes xrandr
 Cflags: -I${includedir}
 Libs: -L${libdir} -lXpresent
-- 
2.6.1

